cypress/integration/AnnualPlan/APSpecialReportShareOption.feature

```diff
@@ -0,0 +1,34 @@
+Feature: function test for share options on special report template for Annual Plan
+
+    Scenario: User views the AP landing page
+        Given user is navigating to "/special-report"
+        And screen breakpoint is set to "desktop"
+        Then the page options should appear in a fixed position to the right of the main content area
+        And the following page options are displayed
+            | breakpoint | pageOption |
+            | desktop    | Facebook   |
+            | desktop    | Twitter    |
```

Comment:
Breakpoint column seems unnecessary - it repeats the same thing. Please remove it, as well as for other scenarios. Also there is no use case for Spanish page  - you can change the tablet scenario to be Spanish Tablet, without adding a separate one